lib/human_player.py

```diff
@@ -1,3 +1,19 @@
 class HumanPlayer:
-    def __init__(self, mark):
+    def __init__(self, ui, board, mark):
+        self.ui = ui
+        self.board = board
         self.mark = mark
+
+    def move(self):
+        self.ui.display("Choose an open space between 1 and 9:")
+        input_number = self.ui.get_user_input()
+        if (
+            input_number.isnumeric()
+            and self.board.valid_input(input_number)
+            and not self.board.is_position_taken(input_number)
+        ):
+            self.board.mark_position(input_number, self.mark)
+            self.board.display_board()
+        else:
+            self.ui.display("Invalid choice!Try again")
```

Comment:
Nit: could you please add a space between the two sentences?

Reply:
done